variables.tf

```diff
@@ -105,6 +105,16 @@ variable "ecs_import_schools_task_definition_file_path" {
   default = "./import_schools_task_definition.json"
 }
 
+variable "ecs_vacancies_scrape_task_definition_file_path" {
+  description = "Task definition to scrape vacancies"
+  default = "./vacancies_scrape_task_definition.json"
+}
+
+variable "vacancies_scrape_schedule_expression" {
+  description = "vacancies_scrape schedule expression"
+  default = "rate(60 minutes)"
```

Comment:
It would be really good to point users to the documentation behind the accepted values here in the description.

If I wanted to change this value to every day I'd like to be confident in a small change without having to understand the ins and outs of our Terraform code to learn that this value eventually gets received by CloudWatch, and it's that API I need to look at.

Reply:
Yep I'll stick that in the description